sharding/nodesCoordinator/numberOfShardsComputer.go

```diff
@@ -14,6 +15,9 @@ func (nsc *numberOfShardsWithMetaComputer) ComputeNumberOfShards(config *epochNo
 	if nbShards < 2 {
 		return 0, ErrInvalidNumberOfShards
 	}
+	if _, ok := config.eligibleMap[core.MetachainShardId]; !ok {
```

Comment:
Is this backwards compatible?

Reply:
I know this couldn't happen on mainnet, but still...

Reply:
yes... it should be backwards compatible, there have been always 3 shards + meta